app/controllers/bananas_controller.rb

```diff
@@ -19,6 +22,9 @@ def index
       test5 = User.first(:conditions => ("username = '#{params[:user]}'"))  
       # add security issue line / unused variable
       test8 = User.first(:conditions => ("username = '#{params[:user]}'"))  
+      # NOTE: This is waste code, add security issue line / unused variable
+      test8 = User.first(:conditions => ("username = '#{params[:user]}'"))  
```

Comment:
__[development][rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `test8`. Did you mean `test`? [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->